### ISSUE TITLE
Banner auto refresh is not triggered when loading AdMob 8.x ads

### ISSUE THREAD

**Issue by sgabello** [mopub/mopub-ios-sdk#383]
One line summary of the issue here

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### Steps to reproduce the behavior:
- Create a new project
- add MoPub SDK
- add AdMob SDK
- add AdMob Network Adapter
- setup refresh interval in MoPub UI

#### Expected behavior:
Ad auto refresh after loading AdMob ad.

#### Observed behavior:
Ad does not auto refresh after loading AdMob ad.

#### Evidence:
Please note that this is not a new project and no settings have been altered. The app worked correctly up to MoPub SDK 5.15.0 and AdMob SDK 7.92. The bug presented itself only after updating to the latest frameworks.
I was able to replicate the issue on a new empty project:
- ads refresh correctly with the debug Ad Unit ID.
- ads DO NOT refresh with a production Ad Unit ID which loads AdMob.

Potentially the issue could be related to the generic load of third party ads (not only AdMob) vs MoPub's own ads, but atm I tested only AdMob.

I can share the barebone Xcode project if necessary: please provide an email address as the project contains my production Ad unit IDs and I don't want to share them on a public page.

This is the debug info generated with the live Ad Unit ID which loads the AdMob Ad:
```	[MoPub][MPConsentManager] Consent changed to dnt from unknown; PII cannot be collected. Reason: Limit ad tracking was enabled and consent implicitly denied by the user
	[MoPub][MPConsentManager] Consent state changed to dnt: Limit ad tracking was enabled and consent implicitly denied by the user
	[MoPub][MPConsentManager] Attempting to synchronize consent state
	[MoPub][MPConsentManager] Ad unit used for GDPR sync: (redacted)
	[MoPub][MPViewabilityManager] Initialized OM SDK version: 1.3.4-Mopub
	[MoPub][MPConsentSyncSerialNetworkSession] Setting URL request as pending: https://ads.mopub.com/m/gdpr_sync
	[MoPub][MPConsentSyncSerialNetworkSession] Determining if next request should be executed.
	[MoPub][MPConsentSyncSerialNetworkSession] Pending request https://ads.mopub.com/m/gdpr_sync is not a duplicate of last completed request. Will execute.
	[MoPub][MPConsentManager] Consent synchronization triggered by application foreground.
	[MoPub][MPConsentManager] Attempting to synchronize consent state
	[MoPub][MPConsentManager] Ad unit used for GDPR sync: (redacted)
	[MoPub][MPConsentSyncSerialNetworkSession] Setting URL request as pending: https://ads.mopub.com/m/gdpr_sync
	[MoPub][MPConsentSyncSerialNetworkSession] Determining if next request should be executed.
	[MoPub][MPConsentSyncSerialNetworkSession] Request in progress. Waiting to execute until complete.
	[MoPub][MPConsentSyncSerialNetworkSession] Finished request: https://ads.mopub.com/m/gdpr_sync
	[MoPub][MPConsentSyncSerialNetworkSession] Determining if next request should be executed.
	[MoPub][MPConsentSyncSerialNetworkSession] Pending request https://ads.mopub.com/m/gdpr_sync is a duplicate of last completed request https://ads.mopub.com/m/gdpr_sync. Will discard.
	[MoPub][MPConsentManager] Attempting to update consent with new state:
	[MoPub][MPConsentManager] Consent changed to dnt from dnt; PII can be collected.
	[MoPub][MPConsentManager] Consent synchronization complete
	[MoPub][MoPub] SDK initialized and ready to display ads.
	[MoPub][MPBannerAdManager][(redacted)] Attempting to load ad
	[MoPub][MPAdServerCommunicator] Requesting an ad from Adserver: https://ads.mopub.com/m/ad
	[MoPub][MPAdServerCommunicator] Adserver responded with:
	[MoPub][MPAdConfiguration] Could not find adapter class named FacebookBannerCustomEvent
	[MoPub][MPAdConfiguration] Could not find adapter class named FacebookBannerCustomEvent
	[MoPub][MPBannerAdManager] Banner ad view is fetching ad type: admob_native
	[MoPub][MPGoogleAdMobBannerCustomEvent][(redacted)] Adapter MPGoogleAdMobBannerCustomEvent attempting to load ad
	[MoPub][GoogleAdMobAdapterConfiguration] Google Mobile Ads SDK initialized succesfully.
	[MoPub][MPGoogleAdMobBannerCustomEvent][(redacted)] Adapter MPGoogleAdMobBannerCustomEvent successfully loaded ad
	[MoPub][MPGoogleAdMobBannerCustomEvent][(redacted)] Adapter MPGoogleAdMobBannerCustomEvent attempting to show ad
	[MoPub][MPBannerAdManager][(redacted)] Ad loaded
	[MoPub][MPAdServerCommunicator] Successfully sent after load URL: (redacted)
```

This is the debug info generated with the debug Ad Unit ID which loads only MoPub's ads:
```
	[MoPub][MPConsentManager] Attempting to synchronize consent state
	[MoPub][MPConsentManager] Consent synchronization complete: GDPR not applicable, consent synchronization will complete immediately
	[MoPub][MPViewabilityManager] Initialized OM SDK version: 1.3.4-Mopub
	[MoPub][MPConsentManager] Consent synchronization triggered by application foreground.
	[MoPub][MPConsentManager] Attempting to synchronize consent state
	[MoPub][MPConsentManager] Consent synchronization complete: GDPR not applicable, consent synchronization will complete immediately
	[MoPub][MoPub] SDK initialized and ready to display ads.
	[MoPub][GoogleAdMobAdapterConfiguration] Google Mobile Ads SDK initialized succesfully.
	[MoPub][MPBannerAdManager][0ac59b0996d947309c33f59d6676399f] Attempting to load ad
	[MoPub][MPAdServerCommunicator] Requesting an ad from Adserver: https://ads.mopub.com/m/ad
	[MoPub][MPAdServerCommunicator] Adserver responded with:
	[MoPub][MPBannerAdManager] Banner ad view is fetching ad type: html
	[MoPub][MPHTMLBannerCustomEvent][0ac59b0996d947309c33f59d6676399f] Adapter MPHTMLBannerCustomEvent attempting to load ad
	[MoPub][MPHTMLBannerCustomEvent] Viewability tracker (0x600000b8eed0) created
	[MoPub][MPOpenMeasurementTracker] Viewability tracker (0x600000b8eed0) added friendly obstruction: Close Button
	[MoPub][MPOpenMeasurementTracker] Viewability tracker (0x600000b8eed0) added friendly obstruction: Overlay
	[MoPub][MPOpenMeasurementTracker] Viewability tracker (0x600000b8eed0) started session
	[MoPub][MPAdWebViewAgent] Loading MoPub URL: mopub://finishLoad
	[MoPub][MPHTMLBannerCustomEvent][0ac59b0996d947309c33f59d6676399f] Adapter MPHTMLBannerCustomEvent successfully loaded ad
	[MoPub][MPOpenMeasurementTracker] Viewability tracker (0x600000b8eed0) ad event: loaded
	[MoPub][MPBannerAdManager][0ac59b0996d947309c33f59d6676399f] Ad loaded
	[MoPub][MPAnalyticsTracker] Tracking impression:(redacted)
	[MoPub][MPOpenMeasurementTracker] Viewability tracker (0x600000b8eed0) ad event: impression
	[MoPub][MPBannerAdManager] Scheduled the autorefresh timer to fire in 30.0 seconds (0x6000003c84b0).
	[MoPub][MPAdServerCommunicator] Successfully sent after load URL:(redacted)
```

You'll notice that the line `[MoPub][MPBannerAdManager] Scheduled the autorefresh timer to fire in 30.0 seconds (0x6000003c84b0).` is not fired on the first debug report.

**Comment by chauduyphanvu** [comment]
@sgabello, thanks for the report. This issue will be addressed today!

Vu Chau
MoPub Team

**Comment by chauduyphanvu** [comment]
Fixed in https://github.com/mopub/mopub-ios-mediation/commit/9de08828c1e057ebf689c673b4ed76efab1317f7. Thanks!
